tad/services/storage.py

```diff
@@ -0,0 +1,32 @@
+from abc import ABC, abstractmethod
+from pathlib import Path
+
+from yaml import dump
+
+
+class WriteService(ABC):
```

Comment:
you should propably add a close function

Reply:
As we indent on for example line 22 in this script, Python closes of automatically right? Or you want this to be more generic so that it work also for other types of storage system like s3 or a database?

Reply:
I want it to be more generic.